tools-v2/README.md

```diff
@@ -1484,6 +1485,27 @@ Output:
 +------+--------------------------------------+--------------------------------------+----------+-------+----------+--------+----------+--------+----------+---------------------+
 ```
 
+##### query snapshot
+
+query snapshot in curvebs
+
+Usage:
+
+```shell
+curve bs query snapshot --path /test/test111 --snapshotstatus done
+(other status: "in-progess" "deleting" "errorDeleteing" "canceling" "error")
```

Comment:
Maybe `##` is better than `()`.

Reply:
Thanks for your comment, but I don't understand the ##, any example?

Reply:
for example, 
```shell
curve bs query snapshot --path /test/test111 --snapshotstatus done # other status: "in-progess" "deleting" "errorDeleteing" "canceling" "error"
```

Reply:
It looks like `done` is a keyword in shell, is that a problem？

Reply:
It seems to be fine in the tests, and done should work only with for or while etc.